src/index.ts

```diff
@@ -105,8 +107,19 @@ export {
   getSpendingLimitsPolicy,
   getSudoPolicy,
   getUniversalActionPolicy,
-  encodeSmartSessionSignature,
+  encodeUseSmartSessionSignature,
+  encodeEnableSmartSessionSignature,
+  isSessionEnabled,
   SmartSessionMode,
+  Session,
+  SessionEIP712,
+  PolicyData,
+  ERC7739Data,
+  ActionData,
+  EnableSession,
+  EnableSessionData,
+  ChainDigest,
+  ChainSession,
```

Comment:
I think we're missing `hashchainsessions`

Reply:
Added